### PR TITLE
Issue #6207: Add Xpath Regression Test for IllegalImport

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Partially Fixes Issue #6207 Add Xpath Regression Test for IllegalImport

### CODE REVIEW

**Comment by Vyom-Yadav** [comment]
@Rahulkhinchi03 You have to rewritte your commit message, it is missing a space after `:`

**Comment by Vyom-Yadav** [comment]
Your commit message is still not correct:
```
Issue #6207: Add Xpath Regression Test for IllegalImport
git commit -m Issue
```
it should be:
```
Issue #6207: Add Xpath Regression Test for IllegalImport
```
You added the git command in the message somehow.

**Comment by romani** [comment]
Make sure `mvn verify ` passing on your local.

CI is red, please fix

**Comment by Rahulkhinchi03** [comment]
GitHub, rebase

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        final DefaultConfiguration moduleConfig =
+                createModuleConfig(IllegalImportCheck.class);
+        moduleConfig.addProperty("illegalPkgs", "java.util");
+        final String[] expectedViolation = {
+            "3:1: " + getCheckMessage(IllegalImportCheck.class,
+                        IllegalImportCheck.MSG_KEY, "java.util.List"),
+        };
+        final List<String> expectedXpathQueries = Collections.singletonList(
+                "/COMPILATION_UNIT/IMPORT"
+        );
```
Comment (romani) [inline src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java:53]: Please change input to be different and xptlath to be different

Use static import.

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java (hunk 1 above)
Comment (Rahulkhinchi03) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java:55]: Changes Done

**Comment by Rahulkhinchi03** [comment]
GitHub, rebase

**Comment by Rahulkhinchi03** [comment]
GitHub, rebase

**Comment by Rahulkhinchi03** [comment]
GitHub, rebase

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java (hunk 1 above)
Comment (romani) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java:53]: One input should be static, second is reguy import

Comment (Rahulkhinchi03) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalImportTest.java:53]: @romani  Changes Done

**Comment by Rahulkhinchi03** [comment]
GitHub, rebase

**Review by romani — APPROVED** [review]

**Pull request merged**